cargo-apk/README.md

```diff
@@ -104,3 +104,7 @@ value = "true"
 ```
 
 TODO: intent filters
+
+## Dynamically loaded libraries
+
+The `build` command scans for extra shared libraries intended to be dynamically loaded with `dlopen`. Files with extension `.so` placed under `lib/${androideabi}` at the root of the local package are added to the apk automatically.
```

Comment:
```suggestion
The `build` command scans for extra shared libraries intended to be dynamically loaded with `dlopen`. Files with extension `.so` placed under `lib/${android_abi}` at the root of the local package are added to the apk automatically.
```

Don't think we need to constrain to Embedded ABI, which is generally only used in the 32-bit ARM triple anyway.